ndk-build/src/apk.rs

```diff
@@ -96,7 +120,56 @@ impl<'a> UnalignedApk<'a> {
         let lib_path = Path::new("lib").join(abi).join(path.file_name().unwrap());
         let out = self.config.build_dir.join(&lib_path);
         std::fs::create_dir_all(out.parent().unwrap())?;
-        std::fs::copy(path, out)?;
+
+        match self.config.strip {
+            StripConfig::Default => {
+                std::fs::copy(path, out)?;
+            }
+            StripConfig::Strip | StripConfig::Split => {
+                let obj_copy = self.config.ndk.toolchain_bin("objcopy", target)?;
+
+                timeit!(format!("strip debuginfo from {}", lib_path.display()), {
+                    let mut cmd = Command::new(&obj_copy);
+                    cmd.current_dir(&self.config.build_dir);
+                    cmd.arg("--strip-debug");
+                    cmd.arg(path);
+                    cmd.arg(&lib_path);
```

Comment:
Does this need to run inside the build dir, can't it simply use the absolute `out` path instead, similar to what's passed to `std::fs::copy()` above?

Same question below.